v0/destinations/zendesk/transform.js

```diff
@@ -348,14 +408,27 @@ async function processIdentify(message, destinationConfig, headers) {
   const returnList = [];
 
   const traits = getFieldValueFromMessage(message, "traits");
+  const userId = await getUserId(message, headers);
+  const primaryEmail = get(traits, "primaryEmail");
+  if (userId && primaryEmail) {
+    const userIdentityId = await getUserIdentityId(
+      userId,
+      primaryEmail,
+      headers
+    );
+    if (userIdentityId) {
+      returnList.push(
+        responseBuilderToSetPrimaryAccount(userIdentityId, userId, headers)
```

Comment:
Don't we need to verify the emails?

Reply:
No, I've checked that, without verification also it lets me add it as primary.